plat/intel/soc/stratix10/plat_sip_svc.c

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <debug.h>
+#include <mmio.h>
+#include <runtime_svc.h>
+#include <uuid.h>
+#include <soc/s10_mailbox.h>
+#include <console_printf.h>
```

Comment:
Where is this ``console_printf.h`` header? It looks like this C file isn't compiled.

Reply:
My bad, some stray codes from when debugging.